services/user-service/migrations/20240201070125-create_table_user.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
I think we can skip the migrations files, just put `.gitkeep` file in here, we will create our own cli later to generate this file later